modules/openy_gc_auth/modules/openy_gc_auth_personify/src/Controller/PersonifyAuthController.php

```diff
@@ -212,25 +215,51 @@ public function signOutUrl(Request $request) {
    * @throws \GuzzleHttp\Exception\GuzzleException
    */
   private function userHasActiveMembership($token) {
+
     $personifyID = $this->personifySSO->getCustomerIdentifier($token);
     if (empty($personifyID)) {
       return FALSE;
     }
 
-    $memberships = $this->personifyClient->doAPIcall(
-      'GET',
-      'OrderMembershipInformationViews?$filter=ShipMasterCustomerId%20eq%20%27' . $personifyID . '%27&$format=json'
-    );
-    if (!isset($memberships['d'])) {
-      return FALSE;
-    }
+    $body = '<StoredProcedureRequest>
+    <StoredProcedureName>YSV_OpenY_Member_Access</StoredProcedureName>
```

Comment:
So now only Silicon Valley can do SSO?

Reply:
This was discussed between You, me and SV team that we should have the same approach in the checking of membership status by using a custom procedure everywhere.
We Can change it to the OpenY_Member_Access, but in general it was approved as new approach for checking.
If this not a strategy for now, we have to understand how to organize this check.

Reply:
@anpolimus could you share SQL procedure in README then?
I'm fine with it, just wanted to make it flexible

Reply:
I don't have that SQL. I was introduced to API request details only.
I'll ask YMCASV team to share it.

Reply:
I've asked them in the letter.